command/commands.go

```diff
@@ -206,6 +207,11 @@ var (
 	initCommandsEnt = func(ui, serverCmdUi cli.Ui, runOpts *RunOptions) {}
 )
 
+func init() {
```

Comment:
This *seemed* like the best place to put this, since it will always be run by the server when starting up, but I'm open to suggestion.

Reply:
You could invoke this in `core.postUnseal`, but perhaps @ncabatoff has a better suggestion.

Reply:
I was trying to target somewhere that will also be run when Agent starts up. (Ideally, also when any plugin starts up in a separate process, but that would require hooking into `sdk/` I think.)

Reply:
This seems perfectly fine to me. The other obvious place is `main.go` in the root of the repo, but maybe I'm missing a reason that wouldn't work.

I think you could also just put the `init` function directly in the internal module and include it as an unnamed import (`_ "github.com/hashicorp/vault/internal"`). That would net a couple of benefits:
* Eliminates all exported API surface to prevent other calls
* Go's `init` function semantics will handle ensuring the function is only ever called once, and before everything else runs, so e.g. you can do away with the `sync.Once` usage.